src/Hangman/Application/CommandHandlers/StartNewGame.handler.ts

```diff
@@ -0,0 +1,23 @@
+import { CommandHandler, ICommandHandler, EventPublisher } from '@nestjs/cqrs';
+import { StartNewGameCommand } from '../Commands/StartNewGame.command';
+import { Logger } from '@nestjs/common';
+import { GamesRepository } from 'src/Hangman/Domain/Repositories/GamesRepository';
+
+@CommandHandler(StartNewGameCommand)
+export class StartNewGameCommandHandler
+  implements ICommandHandler<StartNewGameCommand> {
+  constructor(
+    private readonly repository: GamesRepository,
+    private readonly publisher: EventPublisher,
+  ) {}
+
+  async execute(command: StartNewGameCommand) {
+    Logger.log(command, 'StartNewGameCommandHandler');
+
+    const { gameDto } = command;
+    const game = this.publisher.mergeObjectContext(
+      await this.repository.startGame(gameDto),
```

Comment:
Ik zou hier uitschrijven

```
await this.repository.startGame(gameId, playerId, wordToGuess, maxGuesses)
```

Reply:
but `startGame` needs data to set on it's game, right?